vMenu/EventManager.cs

```diff
@@ -284,11 +284,6 @@ private async Task UpdateWeatherParticles()
         /// <returns></returns>
         private async Task WeatherSync()
         {
-            if (MainMenu.WeatherOptionsMenu == null)
```

Comment:
This would be better suited to a separate PR, since it's unrelated to vehicle extras 👍